File: test/fixtures/javascript-jquery/get_with_form.js
const form = new FormData();
form.append('from', 'test@tester.com');
form.append('to', 'devs@tester.net');
form.append('subject', 'Hello');
form.append('text', 'Testing the converter!');

$.ajax({
  url: 'http://localhost:28139/v3',
  crossDomain: true,
  method: 'post',
  username: 'test',
  password: '',
  data: form
}).done(function(response) {
  console.log(response);
});

File: test/fixtures/java-unirest/get_with_form.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.post("http://localhost:28139/v3")
            .basicAuth("test", "")
            .field("from","test@tester.com")
            .field("to","devs@tester.net")
            .field("subject","Hello")
            .field("text","Testing the converter!")
            .asString();
    }
}
